Middlewares/ST/STM32_USB_Device_Library/Class/video/Inc/usbd_uvc.h

```diff
@@ -132,6 +133,11 @@ typedef enum _vs_terminal_id {
   VC_CONTROL_XU_LEP_RAD_ID,
   VC_CONTROL_XU_LEP_SYS_ID,
   VC_CONTROL_XU_LEP_VID_ID,
+  VC_CONTROL_XU_LEP_AGC_2_ID,
+  VC_CONTROL_XU_LEP_OEM_2_ID,
+  VC_CONTROL_XU_LEP_RAD_2_ID,
```

Comment:
Do you think it would make sense to make this:

```
VC_CONTROL_XU_LEP_RAD_2_ID = VC_CONTROL_XU_LEP_RAD_ID + 0x10,
VC_OUTPUT_TERMINAL_ID = 0x100
```

This will leave the padding for additional modules, and allow us to extend to `id + 0x20` for `*_3`, and so on through `*_10`.

Also, this way you can leave off all the extra IDs unless they are implemented later.

Reply:
Does my commit resolve? Note that 0x100 is not an option, because the ID is a 1 byte value.

Reply:
Ah yes, good catch. It looks good to me.

Reply:
I didn't catch it at first. But VLC complained, so I checked usbview and the ID was showing as 0.